api/src/main/java/ai/djl/ndarray/index/NDIndex.java

```diff
@@ -335,6 +336,11 @@ private int addIndexItem(String indexItem, int argIndex, Object[] args) {
         if (!m.matches()) {
             throw new IllegalArgumentException("Invalid argument index: " + indexItem);
         }
+        // None
```

Comment:
Can you add an example of this to the Javadoc for the NDIndex constructor?

Reply:
Yeah will do when adding ndtest